Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/logicalStructure.xhtml

```diff
@@ -17,6 +17,7 @@
         xmlns:h="http://xmlns.jcp.org/jsf/html"
         xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
         xmlns:p="http://primefaces.org/ui">
+    <!--@elvariable id="readOnly" type="java.lang.Boolean"-->
```

Comment:
What is this comment for, or is it accidentally?

Reply:
This annotation helps IDEs to identify the type of Java variables in EL expressions so it can better determine available properties and methods of those variables. 

We already use this type of annotation in many other Facelets.

Reply:
Yes, I've seen it several times and always wondered what kind of note that should be. ... There is a lot that is good in Eclipse, but for JSP & EL, it only provides poor support.